### ISSUE TITLE
Can't import default export from ESM from basic example

### ISSUE THREAD

**Issue by stevenfitzpatrick** [standard-things/esm#31]
node v : 8.30
npm v : 5.3.0
std/esm : 0.2.2

ESM File (helpers.mjs)

Just trying a default and named export, testing if either would work.

```
export const NAMED_SHOW_ME = 'Named Show Me';

const SHOW_ME = 'SHOW ME';
export default SHOW_ME;
```
index.js

```
require('@std/esm');

const helper = require('./helpers.mjs');
console.log(require('./helpers.mjs'));
console.log(require('./helpers.mjs').default);
console.log(helper);
```

It returns

{}
undefined
{}

**Comment by charlike** [comment]
#15

**Comment by stevenfitzpatrick** [comment]
Isn't this the most basic thing this library is meant for ? I am obviously not understanding something.

**Comment by charlike** [comment]
Yep, you are not alone. :D We just should wait :) It's normal. I'm watching it since the beginning.

**Comment by jdalton** [comment]
Fixed with v0.3.0 🎉
